code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -224,6 +224,7 @@
 		O.emp_act(severity)
 
 /mob/living/carbon/electrocute_act(shock_damage, source, siemens_coeff = 1, safety = 0, override = 0, tesla_shock = 0, illusion = 0, stun = TRUE)
+	SEND_SIGNAL(src, COMSIG_LIVING_ELECTROCUTE_ACT, shock_damage, siemens_coeff, illusion)

```

Comment:
If you want to fix this you need to make it so parent gets called instead, just adding the same signal again is very bad

Reply:
Alright, there we go. This should fix it properly